src/starks/constraints/evaluator.rs

```diff
@@ -5,7 +5,12 @@ use lambdaworks_math::{
     traits::ByteConversion,
 };
 
-#[cfg(debug_assertions)]
+#[cfg(feature = "parallel")]
+use rayon::prelude::{
+    IndexedParallelIterator, IntoParallelIterator, IntoParallelRefIterator, ParallelIterator,
+};
+
+#[cfg(all(debug_assertions, not(feature = "rayon")))]
```

Comment:
seems like it should be `parallel` instead of `rayon` :) 

also check the others